cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/DataPrepBrowserTopPanel/index.tsx

```diff
@@ -23,7 +23,7 @@ require('./DataPrepBrowserTopPanel.scss');
 interface IDataprepBrowserTopPanel {
   allowSidePanelToggle: boolean;
   toggle: (e: React.MouseEvent<HTMLElement>) => void;
-  browserTitle: string;
+  browserTitle: React.ReactNode | string;
```

Comment:
If we are using `ReactNode` don't think `string` is required. `string` is already part of `ReactNode` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L152